### PR TITLE
Add Code Coverage

### PR DESCRIPTION
**Pull request by hayesgm**
This patch adds code coverage to the Open Price Feed system. The coverage will help us make sure we properly cover all cases. I am not currently sure how well this system will work with our immutables, but we'll see. Additionally, this may require extra resources from CircleCI since coverage is significantly more intense (we effectively trace all transactions).

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -49,6 +52,63 @@ jobs:
       - store_artifacts:
           path: ~/junit-oracle
 
+  test_coverage:
+    docker:
+      - image: circleci/node:11
```
Comment (jflatow) [inline .circleci/config.yml:57]: Should we bump node now?

**Comment by jflatow** [comment]
Looks like you rebased and grabbed 2/3 of the commits on `uniswap-polish`, is that intended? If we are going to merge both we probably just want your last commit

File: .circleci/config.yml [hunk 2]
```diff
@@ -41,14 +42,68 @@ jobs:
...
 
+  test_coverage:
+    docker:
+      - image: circleci/node:13
+    working_directory: ~/repo
+    steps:
+      - run:
+        |
+          sudo wget https://github.com/ethereum/solidity/releases/download/v0.6.10/solc-static-linux -O /usr/local/bin/solc
+          sudo chmod +x /usr/local/bin/solc
```
Comment (maxwolff) [inline .circleci/config.yml:59]: we already grabbed solc in `test-oracle`, do we have to re download it? does everything get wiped between jobs?

Comment (hayesgm) [reply .circleci/config.yml:59]: yeah, they run totally separately

**Comment by hayesgm** [comment]
Also FYI, the majority of the real changes occurred here: https://github.com/compound-finance/0x-monorepo/commit/ebceeab03b86468e0cb2b3919482c43413fd88a4

**Pull request merged**